daemon/daemon.go

```diff
@@ -744,6 +745,7 @@ func NewDaemon(config *Config, registryService *registry.Service) (daemon *Daemo
 }
 
 func (daemon *Daemon) Shutdown() error {
+	daemon.shutdown = true
```

Comment:
If we are going to block stops from API when shutdown is happening then I think this is out of scope for this PR. There is much more other logic that shouldn't happen when when daemon is shutting down. Ideally we should block all those requests(and wait for ongoing ones to finish). If this is the goal then for this PR I would much rather drop this shutdown logic and change the `Stop(seconds)` method to `Stop(seconds, isManual)`.